saos-persistence/src/main/java/pl/edu/icm/saos/persistence/repository/JudgmentRepositoryCustom.java

```diff
@@ -24,6 +25,14 @@
      * the saos-enrichment module.
      */
     public <T extends Judgment> T findOneAndInitialize(long id);
+    
+    /**
+     * Counts number of referencing judgments for not indexed judgments.
+     * 
+     * @return map containing pairs of judgmentId and number of
+     *     judgments referencing to judgment with that id
+     */
+    public Map<Long, Long> countReferencingJudgmentsForNotIndexed();
```

Comment:
When when I look at it I think it would be better to have it in a separate service. If it was something like: countReferencingJudgments(judgmentId) it would be ok, but now it is a kind of index functionality involved here. I would move it to search module (similarly to what I have done in UploadEnrichmentTagCopier). What do you think about  it?